### PR TITLE
Added nimcat

### PR DESCRIPTION
**Pull request by shakna-israel**

### CODE REVIEW

**Comment by flaviut** [comment]
Please removing the trailing comma and resubmit. Thanks!

**Comment by dom96** [comment]
Why not just remove the trailing comma yourself?

**Comment by dom96** [comment]
Please get rid of your revert and remove the trailing comma.

**Comment by flaviut** [comment]
> Why not just remove the trailing comma yourself?

That's a good point! Mostly because I've been away so long that I've forgotten how to use GitHub. :wink:

**Comment by shakna-israel** [comment]
Which comma here is the issue?

File: packages.json [hunk 1]
```diff
@@ -3485,5 +3485,17 @@
...
+    "name": "nimcat",
+    "url": "https://github.com/shakna-israel/nimcat",
+    "method": "git",
+    "tags": [
+      "cat",
+      "cli"
+    ],
+    "description": "An implementation of cat in Nim",
+    "license": "MIT",
+    "web": "https://github.com/shakna-israel/nimcat",
```
Comment (dom96) [inline packages.json]: The one at the end here.

**Pull request merged**